nimbleparse/src/main.rs

```diff
@@ -130,18 +130,20 @@ fn main() {
         Ok(x) => x,
         Err(s) => {
             let nlcache = NewlineCache::from_str(&yacc_src).unwrap();
-            if let Some((line, column)) =
-                nlcache.byte_to_line_num_and_col_num(&yacc_src, s.span.start())
-            {
-                writeln!(
-                    stderr(),
-                    "{}: {} at line {line} column {column}",
-                    &yacc_y_path,
-                    &s
-                )
-                .ok();
-            } else {
-                writeln!(stderr(), "{}: {}", &yacc_y_path, &s).ok();
+            for s in s {
```

Comment:
Again, `s` probably isn't the right variable name for `Err(s)` any more. Presumably `Err(errs)` or something like that might make more sense?

Reply:
Argh, I meant to go and fix these after getting it all working, but forgot :(

I figured since we're touching every line here, it's also good to rename `s` to `e`,
it seems like maybe it used to be a string at some point.

fixed in edc3d0fbeb9f3711ca3380cef0ea24f4bd4ba660